web-animations-1/Overview.bs

```diff
@@ -708,6 +701,35 @@ animation model to change, but these operations in themselves do not create
 a new [=animation frame=], rather they merely update the current <a>animation
 frame</a>.
 
+### Timeline Phase ### {#timeline-phases}
+
+A [=timeline=] may be in one of four possible 
+<dfn export lt="timeline phase">phases</dfn> for each of which, a non-normative
+description is also provided:
+
+<div class=informative-bg>
+
+:   <dfn export lt="timeline inactive phase">inactive</dfn>
+::  A <a>timeline</a> is considered to be <dfn lt="inactive timeline">inactive
+    </dfn> when its <a>time value</a> is <a>unresolved</a>. This behavior is
+    further defined by each type of timeline.
```

Comment:
This still feels really odd to me. It feels like it should be the other way around. When the timeline is inactive it will produce an unresolved time (and then we define the circumstances that make it inactive).

Reply:
Done, I moved the requirements for being inactive to the phase calculation for document timeline.